scripts/cli.py

```diff
@@ -247,6 +250,31 @@ def set_quiet(self, quiet):
     def set_debug(self, debug):
         self.debug_app = debug
 
+    def find_flash_dev(self, vid, pid):
+        """
+        Return device port based on Vendor Id and Product Id.
+        Abort if no board or more than one found.
+        """
+        # Get ports that ma
+        re = self.Constants.USB_HID_RE.format(vid, pid)
+        ports = list(serial.tools.list_ports.grep(re))
+
+        # Abort if no board found
+        if not ports:
+            logging.critical('Error: No target board connected!')
+            logging.info('Make sure the board is connected and programable')
+            exit()
```

Comment:
Provide  exit number less than 0 when there is error

Reply:
I totally agree with you. But, for a matter of pattern keeping, we agreed to keep this behavior and fix all its occurrences in a single future commit.